ipn/ipnlocal/local.go

```diff
@@ -2365,8 +2380,14 @@ func (b *LocalBackend) initOnce() {
 // actually a supported operation (it should be, but it's very unclear
 // from the following whether or not that is a safe transition).
 func (b *LocalBackend) Start(opts ipn.Options) error {
-	b.logf("Start")
+	defer b.settleEventBus()
+	b.mu.Lock()
+	defer b.mu.Unlock()
```

Comment:
I know you know these happen in the right order, but what do you think about making it explicit?

```suggestion
	b.mu.Lock()
	defer func() { b.mu.Unlock(); b.settleEventBus() }()
```